addons/calendar/static/src/views/ask_recurrence_update_policy_hook.js

```diff
@@ -3,10 +3,11 @@
 import { useService } from "@web/core/utils/hooks";
 import { AskRecurrenceUpdatePolicyDialog } from "@calendar/views/ask_recurrence_update_policy_dialog";
 
-export function askRecurrenceUpdatePolicy(dialogService) {
+export function askRecurrenceUpdatePolicy(dialogService, acceptedUpdateTypes) {
     return new Promise((resolve) => {
         dialogService.add(AskRecurrenceUpdatePolicyDialog, {
             confirm: resolve,
+            acceptedUpdateTypes: acceptedUpdateTypes
```

Comment:
if the key and the value have the same name, you don't need to repeat it
```suggestion
            acceptedUpdateTypes,
```